apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -207,6 +219,35 @@ function CourseNewRequestCard({ csrfToken }: { csrfToken: string }): HtmlValue {
               not need to provide this if you would like to use the online web editor.
             </small>
           </div>
+          <div class="form-group">
+            <label id="cr-referral-source-label">How did you hear about PrairieLearn?</label>
+            <select
+              class="custom-select"
+              name="cr-referral-source"
+              id="cr-referral-source"
+              aria-labelledby="cr-referral-source-label"
+              required
+            >
+              <option value="" disabled selected></option>
+              <option value="I've used PrairieLearn before">I've used PrairieLearn before</option>
+              <option value="Word of mouth">Word of mouth</option>
+              <option value="Web search">Web search</option>
+              <option value="Conference or workshop">Conference or workshop</option>
+              <option value="Publication">Publication</option>
+              <option value="other">Other...</option>
```

Comment:
These were based on suggestions from Mariana and Copilot. I figured the first one was important since a lot of courses are created for users who have used PL before. Are there any other categories we want to have here?

Reply:
These look great to me and will capture the information we need.